src/CosmoShips.sol

```diff
@@ -11,20 +11,29 @@ import "./IAttributeVerifier.sol";
 contract CosmoShips is ERC721, AccessControl, ReentrancyGuard, AttributeEncoder, ERC721Enumerable {
     bytes32 public constant MAINTENANCE_ROLE = keccak256("MAINTENANCE_ROLE");
     bytes32 public merkleRoot;
+    uint256 private _shipIdToMint;
```

Comment:
_nextTokenIdToMint

Reply:
If we change that to uint16, we will end up removing the counter defined in GameLeague.sol, and so, we will end up having many changes, without a strong benefit, especially now that we want to move fast.
Let's optimize later.